docs/standard/parallel-programming/snippets/vb/snippets.vbproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>snippets</RootNamespace>
     <TargetFramework>net5.0</TargetFramework>
+    <StartupObject>Sub Main</StartupObject>
```

Comment:
I believe this is not valid for StartupObject, and will have no effect.

Reply:
It is valid, as it was set from the **Visual Studio** project UI.

Reply:
Yeah confirmed it's valid, didn't know that. I opened https://github.com/dotnet/docs/pull/24038 to be more specific about "where" the Main should be found to avoid the disambiguate it so that it compiles with multiple `Main`s declared.